app.js

```diff
@@ -34,23 +34,49 @@ function resetStats() {
 }
 
 form.addEventListener('submit', (e) => {
+    e.preventDefault();
     // Step 1 - add code to track each submission to the stats
+    const addStat = new FormData(form);
     // Create an object to "model" you statistic like the following:
+    const userData = {
+        
+
+        player: addStat.get('player'),
+        points: parseInt(addStat.get('points')),
```

Comment:
Clever to use parseInt! You can also use `Number()`. It's good to know both, though, because `parseInt()` will only attempt to parse digits and will stop at the first non-digit character. `Number()`, however, will still attempt to convert the entire string value to a number datatype, which can result in unintended results. 

```js
parseInt('300px') // 300
parseInt('43') // 43

Number('300px') // NaN
Number('43') // 43
```

As you can see, `Number()` attempts to convert the entire string input value, but `parseInt()` parses ignore any non-digit characters.